vxyowsup/whatsapp.py

```diff
@@ -154,8 +154,8 @@ def onMessage(self, messageProtocolEntity):
             messageProtocolEntity.getParticipant())
         self.toLower(receipt)
 
-        print "You have received a message, and thusly sent a receipt"
-        print "You are now sending a reply"
+        log.debug('You have received a message, and thusly sent a receipt')
+        # log.debug("You are now sending a reply")
         # self.send_to_human('iiii', from_address + '@s.whatsapp.net', "this transport's gone rogue")
```

Comment:
Actually it seems a bit more intricate than I thought it would be because of the threads. (This should be the tag line of this project, tbh.)  I think I'd need to call `handle_outbound_message` from the thread so that the vumi id goes into redis. So I think I'll put this into a new issue :grin: